kaspresso/src/main/kotlin/com/kaspersky/kaspresso/kaspresso/Kaspresso.kt

```diff
@@ -145,7 +145,8 @@ data class Kaspresso(
     internal val objectBehaviorInterceptors: List<ObjectBehaviorInterceptor>,
     internal val deviceBehaviorInterceptors: List<DeviceBehaviorInterceptor>,
     internal val stepWatcherInterceptors: List<StepWatcherInterceptor>,
-    internal val testRunWatcherInterceptors: List<TestRunWatcherInterceptor>
+    internal val testRunWatcherInterceptors: List<TestRunWatcherInterceptor>,
+    internal val resourcesRootDirsProvider: ResourcesRootDirsProvider
```

Comment:
Maybe it is more useful to have `ResourceFilesProvider` here? If we need only root dirs we still can get them from the files paths.